<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <layout class="QHBoxLayout" name="toolsAsHeaderLayout">
       <item>
        <layout class="QVBoxLayout" name="toolsGroupLayout">
         <item>
          <layout class="QHBoxLayout" name="captureGroupLayout">
           <item>
            <widget class="QLabel" name="captureDeviceLabel">
             <property name="text">
              <string>Capture device file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="captureDeviceFileName">
             <property name="text">
              <string comment="Set if defaul device not working"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="captureDeviceApplyButton">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="captureDeviceDefaultButton">
             <property name="text">
              <string>Reset to default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="brigthnessGroupLayout">
           <item>
            <widget class="QLabel" name="brigthnessThresholdLabel">
             <property name="text">
              <string>Markers threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="brigthnessThresholdSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="graphicsPixmapLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>VideoCapture</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captureDeviceDefaultButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onCaptureDeviceDefaultButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captureDeviceApplyButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onCaptureDeviceApplyButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brigthnessThresholdSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onBrigthnessThresholdSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_buttonBox_rejected()</slot>
  <slot>onCaptureDeviceDefaultButtonClicked()</slot>
  <slot>onCaptureDeviceApplyButtonClicked()</slot>
  <slot>onBrigthnessThresholdSliderMoved(int)</slot>
 </slots>
</ui>
